src/region/RegionAnnotations.tsx

```diff
@@ -74,7 +74,7 @@ export default class RegionAnnotations extends React.PureComponent<Props, State>
             return;
         }
 
-        createRegion({ ...staged, message });
+        createRegion({ ...(staged as CreatorRegion), message });
```

Comment:
Why this type assertion is previously not needed?

Reply:
Because the `CreatorItem` type is now a union of `CreatorRegion | CreatorHighlight`

Reply:
This shouldn't be necessary, since `staged` is already typed as a CreatorRegion, right?